examples/aggregator/config/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
```

Comment:
Could you also source `scripts/common` here, or is it too complicated?

Reply:
It's not, but afaict, these bash script will be there for a very short time (as @ipetr0v  is replacing them with toml files). @ipetr0v, is that correct?

Reply:
Yes, but `app_config_serializer` is the only thing that changes in https://github.com/project-oak/oak/pull/1219.
Examples are still compiled from `scripts/build_example`